doc/content/reference/configuration/application-server.md

```diff
@@ -27,6 +27,18 @@ Application Server exposes an MQTT server for streaming data.
 - `as.mqtt.public-address`: Public address of the MQTT frontend (default "localhost:1883")
 - `as.mqtt.public-tls-address`: Public address of the MQTTs frontend (default "localhost:8883")
 
+## PubSub Options
+
+- `as.pubsub.providers.mqtt`: Controls the status of MQTT provider (default `enabled`)
+- `as.pubsub.providers.nats`: Controls the status of NATS provider (default `enabled`)
+
+## AWS IoT Integration Options
+
+{{< distributions "Cloud" "Enterprise" "AWS Launcher" >}}
+
+- `as.aws.iot.telemetry`: Enable publishing telemetry to AWS IoT (deprecated)
```

Comment:
I don't think we should add deprecated options to the documentation if they weren't already there.